.travis.yml

```diff
@@ -43,6 +43,8 @@ matrix:
       env: DRIVER=ruby REDIS_BRANCH=3.2
     - rvm: "2.3"
       env: DRIVER=ruby REDIS_BRANCH=4.0
+    - rvm: "2.3"
+      env: DRIVER=ruby REDIS_BRANCH=6.0
```

Comment:
It would be better to make 6.0 the default version in test, so that it's tested against all rubies.